libs/partners/prompty/tests/unit_tests/prompts/basic_chat.prompty

```diff
@@ -0,0 +1,28 @@
+---
+name: Basic Prompt
+description: A basic prompt that uses the GPT-3 chat API to answer questions
+authors:
+  - author_1
+  - author_2
+model:
+  api: chat
+  configuration:
+    azure_deployment: gpt-35-turbo
+sample:
+  firstName: Jane
+  lastName: Doe
+  question: What is the meaning of life?
+  chat_history: []
+---
+system:
+You are an AI assistant who helps people find information.
+As the assistant, you answer questions briefly, succinctly, 
+and in a personable manner using markdown and even add some personal flair with appropriate emojis.
+
+{{#chat_history}}
+{{role}}:
+{{content}}
+{{/chat_history}}
```

Comment:
```suggestion
placeholder:
{{chat_history}}
```
Would you consider something like this for chat history in the langchain runtime? If so, we can get rid of the custom parsing and interpret this as a `MessagesPlaceholder` object, which does the same thing as this.

I'm not sure it makes sense to generate mixed sequence messages and iterated contents of messages because it's typically one or the other. Are you imagining a case where users want to prompty-format a generic list of inputs into multiple messages, instead of specifically 
```
role:
content
```
blocks?

Reply:
The reason I ask is because chat histories in the langchain runtime will typically be passed in as a `List[BaseMessage]` instead of `List[Dict]` where dict matches the format `{"role": x, "content": y}`

This is rendered well by `MessagesPlaceholder`, but would not be rendered well for other messages cases (e.g. ones with `tool_calls` or `ToolMessage`).

Reply:
You mean using **placeholder** as a special keyword in prompty? We want prompty to be usable across orchestration framework though (Promptflow, SemanticKernel etc), so I'm reluctant to add framework specific keywords. Maybe we can add a special langchain parser to support that later? But for //build we want to stick to a more generic approach. 

That's also the main reason why I'm not ready to switch from returning Runnable<> to ChatTemplate. Coz then I would need something special in prompty to indicate it should be swapped to MessagesPlaceholder. And that concept doesn't exist in other places we want to support.